Update Set-CsTeamsAppSetupPolicy.md (#10667)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Set-CsTeamsAppSetupPolicy.md

### Example 1

```powershell
$identity = "test"
# Create new teams app setup policy named "Set-Test".

> New-CsTeamsAppSetupPolicy -Identity Set-$identity

Author: thanks for your suggestion,fixed it.

```

### -Identity

> Name of App setup policy. If empty, all Identities will be used by default.

Reviewer: ```suggestion
Name of app setup policy. If empty, all identities will be used by default.
```

Accept wildcard characters: False
```

### -AllowUserPinning

> If you turn this on, the user’s existing app pins will be added to the list of pinned apps set in this policy. Users can rearrange, add, and remove pins as they choose. If you turn this off, the user’s existing app pins will be removed and replaced with the apps defined in this policy

Reviewer: ```suggestion
If you turn this on, the user’s existing app pins will be added to the list of pinned apps set in this policy. Users can rearrange, add, and remove pins as they choose. If you turn this off, the user’s existing app pins will be removed and replaced with the apps defined in this policy.
```

```yaml
Type: Boolean
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -AllowSideLoading
This is also known as side loading. This setting determines if a user can upload a custom app package in the Teams app. Turning it on lets you create or develop a custom app to be used personally or across your organization without having to submit it to the Teams app store. Uploading a custom app also lets you test an app before you distribute it more widely by only assigning it to a single user or group of users.

```yaml
Type: Boolean
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -PinnedAppBarApps

> Pinning an app displays the app in the app bar in Teams client. Admins can pin app and they can allow users to pin. Pinning is a used to highlight apps that are needed the most by users and promote ease of access.

Reviewer: ```suggestion
Pinning an app displays the app in the app bar in Teams client. Admins can pin apps and they can allow users to pin apps. Pinning is used to highlight apps that users need the most and promote ease of access.
```

```yaml
Type: Microsoft.Teams.Policy.Administration.Cmdlets.Core.PinnedApp[]
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -PinnedMessageBarApps
Apps will be pinned in messaging extensions and into the ellipsis menu.

```yaml
Type: Microsoft.Teams.Policy.Administration.Cmdlets.Core.PinnedMessageBarApp[]
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -AppPresetList

> Choose which apps and messaging extensions you want installed in your users' personal Teams environment and in meetings they create. Users can install other available apps from the Teams app store. 

Reviewer: ```suggestion
Choose which apps and messaging extensions you want to be installed in your users' personal Teams environment and in meetings they create. Users can install other available apps from the Teams app store. 
```
